code/_onclick/hud/rendering/plane_master_controller.dm

```diff
@@ -108,6 +108,7 @@ INITIALIZE_IMMEDIATE(/atom/movable/plane_master_controller)
 		GAME_PLANE_FOV_HIDDEN,
 		GAME_PLANE_UPPER,
 		GAME_PLANE_UPPER_FOV_HIDDEN,
+		ABOVE_GAME_TRANSPARENT_PLANE,
```

Comment:
A: this is a really bad name dude
B: you need to make a plane master for this. Just sticking the plane in this list doesn't do anything if we can't take control over it

Reply:
I did add a plane master for it! It’s in plane_master.dm

What about ABOVE_GAME_NO_MOUSE_TRANSPARENT_PLANE ?

Reply:
oh I'm blind
I guess I'm just annoyed by all the GAME_PLANE_LOWER GAME_PLANE_UPPER GAME_PLANE_UPPER_FOV_HIDDEN that fov caused.

I think I prefer it when planes are particular, rather then general. So AREA_PLANE rather then ABOVE EVERYTHING EXCEPT_X_PLANE. it's hard to pin down. maybe i'm just being anal about this. IDK